javascript/javascript1/week2/homework/flight-booking.js

```diff
@@ -0,0 +1,29 @@
+// Exercise 1 - Flight booking fullname function.
+function getFullname(firstName, surName, useFormalName, sex) {
+  if (firstName === "" || surName === "") {
+    return "Please enter your first name and surname.";
+  }
+
+  if (useFormalName && sex.toLowerCase() === "male") {
+    // Use the toLowerCase() method to ensure the input is in lowercase. 
+    return "Lord " + firstName + " " + surName;
+  } else if (useFormalName && sex.toLowerCase() === "female") {
+    return "Lady " + firstName + " " + surName;
+  } else {
+    return firstName + " " + surName;
+  }
+}
+
+// Create two variables and call the function using them.
+const fullname1 = getFullname("Benjamin", "Hughes");
+const fullname2 = getFullname("Benjamin", "Hughes");
+
+// Log the output
+/* console.log(fullname1);
+console.log(fullname2); */
+
+// Test the function with different values and the additional parameter.
+console.log(getFullname("Benjamin", "Hughes", true, "MALE"));
+console.log(getFullname("Ana", "Hughes", true, "female"));
+console.log(getFullname("Benjamin", "Hughes", false));
```

Comment:
If you do call the function like this
```suggestion
console.log(getFullname("Benjamin", "Hughes", true));
```

You'd get an error. See how can you improve the logic to prevent that.
![error](https://github.com/user-attachments/assets/9fbb8f42-9e04-4e20-99cc-13070ed8c145)